mycroft/skills/phillips_hue/__init__.py

```diff
@@ -0,0 +1,283 @@
+# Copyright 2016 Mycroft AI, Inc.
+#
+# This file is part of Mycroft Core.
+#
+# Mycroft Core is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Mycroft Core is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Mycroft Core.  If not, see <http://www.gnu.org/licenses/>.
+
+from adapt.intent import IntentBuilder
+from mycroft.skills.core import MycroftSkill
+from mycroft.util.log import getLogger
+from os.path import dirname
+from phue import Bridge
+from phue import Group
+from phue import PhueRegistrationException
+from phue import PhueRequestTimeout
+import socket
+import time
+
+__author__ = 'ChristopherRogers1991'
+
+LOGGER = getLogger(__name__)
+
+
+class DeviceNotFoundException(Exception):
+    pass
+
+
+class PhillipsHueSkill(MycroftSkill):
+
+    def __init__(self):
+        super(PhillipsHueSkill, self).__init__(name="PhillipsHueSkill")
+        self.brightness_step = self.config.get('brightness_step', '')
+        if self.brightness_step == '':
+            self.brightness_step = 50
+        else:
+            self.brightness_step = int(self.brightness_step)
+        self.color_temperature_step =\
+            self.config.get('color_temperature_step', '')
+        if self.color_temperature_step == '':
+            self.color_temperature_step = 1000
```

Comment:
I think that since we have defaults in `mycroft.ini`, we don't need to have a fallback for when the config is empty.